### PR TITLE
add taskmeta not exist check

### PR DESCRIPTION
**Pull request by RichardXuan**
Task endpoint extra field add taskmeta not exist check prevent query not exist situation

### CODE REVIEW

**Review by rhyw — COMMENTED** [review]
forgot to submit my comment.

**Comment by hanchuntao** [comment]
I don't know where use these `get_status`, `get_date_done`, `get_traceback`, `get_object_url`

**Comment by RichardXuan** [comment]
@hanchuntao It's for SerializerMethodField. You can see this doc for more information https://www.django-rest-framework.org/api-guide/fields/#serializermethodfield.

File: openlcs/tasks/serializers.py [hunk 1]
```diff
@@ -22,19 +22,35 @@ class Meta:
                   'date_done', 'traceback', 'object_url')
 
     def get_status(self, obj):
-        return TaskMeta.objects.get(task_id=obj.meta_id).status
+        try:
+            taskmeta = TaskMeta.objects.get(task_id=obj.meta_id)
+        except TaskMeta.DoesNotExist:
+            return None
```
[the remark this answers is not in the document]
Comment (RichardXuan) [reply openlcs/tasks/serializers.py:28]: Ok. Thanks.

**Comment by hanchuntao** [comment]
> @hanchuntao It's for SerializerMethodField. You can see this doc for more information https://www.django-rest-framework.org/api-guide/fields/#serializermethodfield.

**Comment by RichardXuan** [comment]
@hanchuntao It used in task endpoint to display extra field.
```json
[
    {
        "id": 5,
        "meta_id": "60747ba3-653f-4669-b37e-08eb8b3ae14f",
        "owner": "admin",
        "params": "{"package_nvr": "ansible-2.4.2.0-2.el7", "license_scan": true,
         "copyright_scan": true, "product_release": "satellite-6.9.0"}",
        "status": "SUCCESS",
        "date_done": "2022-11-10T05:39:45.746881",
        "traceback": null,
        "object_url": "https://openlcs-dev.apps.ocp-c1.prod.psi.redhat.com/rest/v1/sources/2/"
    },
    {
        "id": 6,
        "meta_id": "ddc8d40a-3c85-4a14-ace8-440c7e69ad55",
        "owner": "admin",
        "params": "{"package_nvr": "fio-3.1-2.el7", "license_scan": true,
         "copyright_scan": true, "product_release": "satellite-6.9.0"}",
        "status": "SUCCESS",
        "date_done": "2022-11-10T05:38:50.461372",
        "traceback": null,
        "object_url": "https://openlcs-dev.apps.ocp-c1.prod.psi.redhat.com/rest/v1/sources/1/"
    }
]
```

**Comment by hanchuntao** [comment]
Got it, thanks for your clarification, LGTM

**Pull request merged**